core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -257,7 +257,7 @@ sealed abstract class Chunk[+A] extends ChunkLike[A] with Serializable { self =>
     var lastA = null.asInstanceOf[A]
 
     foreach { a =>
-      if (a != lastA) builder += a
+      if (a != lastA) builder.addOne(a)
```

Comment:
In some cases it helps, but there maybe not. So I decided replace it everywhere in this file because I don't want to skip correct place